RELEASE

```diff
@@ -18,7 +18,7 @@ Write release entries:
 2. Commit changes to git:
 
 # MAKE SURE THIS IS CORRECT
-export release_ver="1.9.27"
+export release_ver="1.9.28"
```

Comment:
Won't this make tagging awkward? You need to set the tag retroactively.

Reply:
I forgot to do this just after tagging 1.9.27 -- the next release is 1.9.28

Reply:
Oh I see, OK.